sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -865,6 +1052,10 @@ public TableReference getTable() {
       return toBuilder().setCoder(coder).build();
     }
 
+    public TypedRead<T> withReadSessionOptions(ReadSessionOptions options) {
```

Comment:
Document this, and generally add documentation at class level about the availability of two methods and how to choose between them?